src/Debug.h

```diff
@@ -132,6 +135,11 @@ void ResyncDebugLog(FILE *newDestination);
         } \
    } while (/*CONSTCOND*/ 0)
 
+
+/// Stream manipulator to force current debugs to be printed to syslog
+/// at LOG_ALERT level.
```

Comment:
We do not need to say that a manipulator is a manipulator. However, let's document this manipulator (unusual) effect on the stream and the fact that this function should not be used outside of debugs():

```
/// Does not change the stream being manipulated. Exists for its side effect:
/// In a debugs() context, forces the message to become a syslog ALERT.
/// Outside of debugs() context, has no effect and should not be used.
```